src/v2/Apps/Auctions/Components/MyBids/MyBidsBidHeader.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import { MyBidsBidHeader_sale } from "v2/__generated__/MyBidsBidHeader_sale.graphql"
+import { Box, CalendarIcon, Flex, Image, Text } from "@artsy/palette"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+
+interface MyBidsBidHeaderProps {
+  sale: MyBidsBidHeader_sale
+}
+
+export const MyBidsBidHeader: React.FC<MyBidsBidHeaderProps> = ({ sale }) => {
+  return (
+    <RouterLink to={`/auction/${sale.slug}`} noUnderline>
+      {sale.coverImage && (
+        <Box minHeight={100}>
```

Comment:
This just refers to px, right? I think it was discussed previously, but did we ever reach a conclusion on if we wanted to only use numbers in styled-system if it matched up with the theme config and otherwise use a string w/ units?

Reply:
I saw `height` and just jumped out of the spacing scale. If it was margin or padding I would have leaned on the theme, though.